src/mlops_codex/preprocessing.py

```diff
@@ -953,21 +958,21 @@ def create(
             Path of the source file. The file must have a scoring function that accepts two parameters: data (data for the request body of the preprocessing) and preprocessing_path (absolute path of where the file is located)
         requirements_file : str
             Path of the requirements file. The packages versions must be fixed eg: pandas==1.0
-        schema : Union[str, dict]
-            Path to a JSON or XML file with a sample of the input for the entrypoint function. A dict with the sample input can be sending as well. Mandatory for Sync preprocessing
         group : str
-            Group the preprocessing is inserted. Default to 'datarisk' (public group)
-        extra_files : list, optional
+            Group the preprocessing is inserted.
+        schema : Optional[Union[str, dict]]
+            Path to a JSON or XML file with a sample of the input for the entrypoint function. A dict with the sample input can be sending as well. Mandatory for Sync preprocessing
```

Comment:
I'm not sure if this is specific for the sync script, but for the new async we're allowing parquet files too

Reply:
https://github.com/datarisk-io/mlops_codex/pull/68/commits/cba378e43e1d8e8edff6c5cf26d98246b96928c5